src/pp_creators/basic_creator.py

```diff
@@ -255,6 +255,7 @@ def energy_consumer_creator(
             node_breaker_network: NetworkService
     ) -> Dict[str, PpElement]:
         p, q = self.ec_load_provider(energy_consumer)
```

Comment:
What is the reason for adding this mapping here? I ask because the convention we have followed in the mapping is to map elements created by each creator with the associated identified object in our CIM model that was used to create them. That topological node is not created here, it is created when processing connectivity nodes and terminals in the topological node creation code unless I am missing something.

Reply:
Honestly this was kind of a hack. I now know the problem is that topological nodes are not including ECs/PECs in their equipment because they are not defined to have negligible impedance.

Reply:
It still feels a bit like a hack, but the purpose of grouping equipment with negligible impedance is to identify all the **lines** with sufficient impedance, as those are the only things in the BBN with impedance (as far as I know).

Reply:
I wouldn't expect PECs or ECs to be included into a topological node. The purpose of collapsing equipment into topological nodes is to remove them if they serve no purpose in the load flow e.g. closed switches. The only things I would expect to be collapsed into topological nodes are switches/jumpers/junctions.

This is the definition of a topological node:
"a topological node is a set of connectivity nodes that, in the current network state, are connected together through any type of closed switches, including jumpers. Topological nodes change as the current network state changes (i.e., switches, breakers, etc. change state)."